src/components/hero/_hero.scss

```diff
@@ -3,7 +3,7 @@
     position: relative;
     width: 100%;
     padding: 4rem 0;
-    min-height: 100px;
+    min-height: 200px;
```

Comment:
This was to fix a weird issue where there was extra whitespace on the bottom of the page.  Totally not sure why it fixes it, or what the underlying issue was.

I'm unlikely to have empty heros in the future so it should not be an issue moving forward.